web/components/feed/feed-comments.tsx

```diff
@@ -71,6 +65,15 @@ export function FeedCommentThread(props: {
             comment={comment}
             betsBySameUser={betsByUserId[comment.userId] ?? []}
             onReplyClick={scrollAndOpenReplyInput}
+            probAtCreatedTime={
```

Comment:
Reasonable-looking hack. Might want a more robust solution in the long-term, or to ax probability displays in the comments entirely. I personally am not sure.